src/d3/scatterplot.d3.spec.ts

```diff
@@ -27,17 +29,19 @@ describe('VR Scatter Plot', () => {
   });
   it('places points for each element in a one element array', () => {
     scatterplot.init(element, scatterPlotData1, MetaType.SCATTER_PLOT);
-    const expectedPosArray = [{ x: 0, y: 0, z: 0 }];
+    const posArray = [{ x: 20, y: 10, z: 5 }];
+    const expectedPosArray = scalePosition(posArray);
     const result = getPosition(element, shape);
     expect(result).toEqual(expectedPosArray);
   });
   it('places points for each element in a eight element array', () => {
     scatterplot.init(element, scatterPlotData8, MetaType.SCATTER_PLOT);
-    const expectedPosArray = [
+    const posArray = [
       { x: 0, y: 0, z: 0 }, { x: 20, y: 10, z: 5 },
       { x: 40, y: 20, z: 10 }, { x: 60, y: 30, z: 15 },
       { x: 80, y: 40, z: 20 }, { x: 100, y: 50, z: 25 },
      { x: 120, y: 60, z: 30 }, { x: 140, y: 70, z: 35 }];
+    const expectedPosArray = scalePosition(posArray);
```

Comment:
Can hard-code expected values directly instead of copying the function to the test file.